### PR TITLE
moved the location of ? in the url regex to catch url query, unskip "E…

### PR DESCRIPTION
**Pull request by bakatzir**
…xtract Indicators From File - Generic v2 - Test"

## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/18795

## Required version of Demisto
4.1.0

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation - no change
- [x] Code Review

### CODE REVIEW

**Comment by bakatzir** [comment]
@kirbles19 RN change(should be the same string in both):
priot to 5.0:
https://github.com/demisto/content/blob/e65bc8fd4f3600032e3b5fd04ab6ed90cbbe6511/Misc/reputations_CHANGELOG.md
5.0:
https://github.com/demisto/content/blob/e65bc8fd4f3600032e3b5fd04ab6ed90cbbe6511/Misc/reputation-url_CHANGELOG.md

File: Tests/scripts/constants.py [hunk 1]
```diff
@@ -30,6 +30,7 @@
 TEST_SCRIPT_REGEX = r"{}.*script-.*\.yml$".format(TEST_PLAYBOOKS_DIR)
 SCRIPT_REGEX = r"{}.*script-.*\.yml$".format(SCRIPTS_DIR)
 INTEGRATION_PY_REGEX = r"{}.*\.py$".format(INTEGRATIONS_DIR)
+INTEGRATION_PY_UNIT_TESTS_REGEX = r"{}.*\_test.py$".format(INTEGRATIONS_DIR)
```
Comment (yaakovi) [inline Tests/scripts/constants.py]: why is that here?
and why do you escape `_`?
